### ISSUE TITLE
Error in date parsing will send too many emails

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cberetta/gmail-delay-send#16]
```
When the user enters a date that is incorrectly formatted (eg 'wednes 2pm') 
they will receive emails every trigger (5 minutes for most users) notifying 
them that there is a problem.

Should be a better way to handle this..
```

Original issue reported on code.google.com by `blairk...@gmail.com` on 27 Nov 2011 at 4:30

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 0.7.4 (sneak peak: 
https://docs.google.com/spreadsheet/ccc?key=0AgKSjMWgSECadDhtVVlOUXFLTmktSUdrZXB
lZjVQeEE)

When an email cannot be parsed by the script another label is created (if 
necessary) then applied (DELAY_SEND_ERROR).

In further invocations of the script any threads that have this label applied 
will be ignored.
```

Original comment by `blairk...@gmail.com` on 3 Dec 2011 at 6:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `blairk...@gmail.com` on 18 Dec 2011 at 9:24
- Changed state: **Fixed**
